CHANGELOG.md

```diff
@@ -1,6 +1,197 @@
 Changelog
 =========
 
+1.0.0
+-----
+
+### Modifier Delimiters ([#76])
+
+Added support to separate value of modifier from name of modifier with specified string.
+
+Before you could only specify a string to separate name of modifier from name of block or element. It string used to separate value of modifier from name of modifier.
```

Comment:
you -> one
of modifier from name of block or element -> of a modifier from name of a block or an element